install/install.php

```diff
@@ -102,6 +102,13 @@ protected function _activate() {
 			$options['version'] = POLYLANG_VERSION;
 		}
 
+		if ( false === get_option( 'pll_language_from_content_available' ) ) {
+			update_option(
+				'pll_language_from_content_available',
+				0 === $options['force_lang'] ? 'yes' : 'no'
+			);
+		}
+
 		$options->save(); // Force save here to prevent any conflicts with another instance of `Options`.
```

Comment:
Just to keep related code code together:
```suggestion
		$options->save(); // Force save here to prevent any conflicts with another instance of `Options`.

		if ( false === get_option( 'pll_language_from_content_available' ) ) {
			update_option(
				'pll_language_from_content_available',
				0 === $options['force_lang'] ? 'yes' : 'no'
			);
		}
```